Plugins/upenn.cbica.captk.example3/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+project(upenn_cbica_captk_example3)
+
+include_directories(${CTK_INCLUDE_DIRS})
+
+mitk_create_plugin(
+	EXPORT_DIRECTIVE UPENN_CBICA_CAPTK_EXAMPLE3
+	MODULE_DEPENDS PRIVATE MitkExampleInverter
```

Comment:
The PRIVATE scope is not needed, this could read "MODULE_DEPENDS MitkExampleInverter" to make it simpler.

Obviously scoping might be better, but I'm not really aware of the usecases in mitk_create_plugin. Could raise issues if someone tries to call functions of the plugin from another plugin?

[obviously not important, so don't spend more than 10sec deciding :P]